tests/unit/helpers/format-html-message-test.js

```diff
@@ -1,83 +1,88 @@
 import formatHtmlHelper from 'ember-intl/helpers/format-html-message';
-import expectError from '../../helpers/expect-error';
 
-moduleForComponent('format-html-message', {
-  integration: true,
-  beforeEach() {
-    this.inject.service('intl');
+module('format-html-message', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.intl = this.owner.lookup('service:intl');
     this.intl.addTranslations('en-us', { foo: { bar: 'foo bar baz', baz: 'baz baz baz' } });
     this.intl.setLocale('en-us');
-  }
-});
+  });
 
-test('exists', function(assert) {
-  assert.expect(1);
-  assert.ok(formatHtmlHelper);
-});
+  test('exists', function(assert) {
+    assert.expect(1);
+    assert.ok(formatHtmlHelper);
+  });
 
-test('invoke the formatHTMLMessage directly', function(assert) {
-  assert.expect(1);
-  assert.equal(
-    this.intl.formatHtmlMessage('<strong>Hello {name} {count, number}</strong>', {
-      name: '<em>Jason</em>',
-      count: 42000
-    }).string,
-    '<strong>Hello &lt;em&gt;Jason&lt;/em&gt; 42,000</strong>'
-  );
-});
+  test('invoke the formatHTMLMessage directly', function(assert) {
+    assert.expect(1);
+    assert.equal(
+      this.intl.formatHtmlMessage('<strong>Hello {name} {count, number}</strong>', {
+        name: '<em>Jason</em>',
+        count: 42000
+      }).string,
+      '<strong>Hello &lt;em&gt;Jason&lt;/em&gt; 42,000</strong>'
+    );
+  });
 
-test('invoke the formatHTMLMessage directly with inlined locale', function(assert) {
-  assert.expect(1);
+  test('invoke the formatHTMLMessage directly with inlined locale', function(assert) {
+    assert.expect(1);
 
-  let output = this.intl.formatHtmlMessage('<strong>Hello {name} {count, number}</strong>', {
-    name: '<em>Jason</em>',
-    count: 42000,
-    locale: 'fr-fr'
-  }).string;
+    let output = this.intl.formatHtmlMessage('<strong>Hello {name} {count, number}</strong>', {
+      name: '<em>Jason</em>',
+      count: 42000,
+      locale: 'fr-fr'
+    }).string;
 
-  assert.equal(
-    escape(output),
-    '%3Cstrong%3EHello%20%26lt%3Bem%26gt%3BJason%26lt%3B/em%26gt%3B%2042%A0000%3C/strong%3E'
-  );
-});
+    assert.equal(
+      escape(output),
+      '%3Cstrong%3EHello%20%26lt%3Bem%26gt%3BJason%26lt%3B/em%26gt%3B%2042%A0000%3C/strong%3E'
+    );
+  });
 
-test('message is formatted correctly with argument', function(assert) {
-  assert.expect(1);
-  this.render(hbs`{{format-html-message (l "Hello {name}") name="Jason"}}`);
-  assert.equal(this.$().text(), 'Hello Jason');
-});
+  test('message is formatted correctly with argument', async function(assert) {
+    assert.expect(1);
+    await render(hbs`{{format-html-message (l "Hello {name}") name="Jason"}}`);
+    assert.equal(this.element.textContent, 'Hello Jason');
+  });
 
-test('should throw if called with out a value', function(assert) {
-  assert.expect(1);
+  test('should throw if called with out a value', async function(assert) {
```

Comment:
debuggers can be removed.  was there an issue with this test and if so want me to look into it?

Reply:
There is a problem, but it's not specific to this project.
It seems that is hard to catch assertions thrown during render. The usual things you'd expect like

```
render(hbs`thing-that-fails`).catch((e) => asserterror);
// or
try {
  await render(hbs`thing-that-fails`);
} catch (e) {
  assertError
}
```

don't work. 
I believe @rwjblue was inventigating how to make that more natural.